aarch64_linux/aarch64_wheel_ci_build.py

```diff
@@ -9,100 +9,275 @@
-    '''
-    print('Building Arm Compute Library')
-    acl_build_flags=["debug=0", "neon=1", "opencl=0", "os=linux", "openmp=1", "cppthreads=0",
-                     "arch=armv8a", "multi_isa=1", "fixed_format_kernels=1", "build=native"]
-    acl_install_dir="/acl"
-    acl_checkout_dir="ComputeLibrary"
+    """
+    print("Building Arm Compute Library")
+    acl_build_flags = [
+        "debug=0",
+        "neon=1",
+        "opencl=0",
+        "os=linux",
+        "openmp=1",
+        "cppthreads=0",
+        "arch=armv8a",
+        "multi_isa=1",
+        "fixed_format_kernels=1",
+        "build=native",
+    ]
+    acl_install_dir = "/acl"
+    acl_checkout_dir = "ComputeLibrary"
+    """
+    print("Building Arm Compute Library")
+    acl_build_flags = [
+        "debug=0",
+        "neon=1",
+        "opencl=0",
+        "os=linux",
+        "openmp=1",
+        "cppthreads=0",
+        "arch=armv8a",
+        "multi_isa=1",
+        "fixed_format_kernels=1",
+        "build=native",
+    ]
+    acl_install_dir = "/acl"
+    acl_checkout_dir = "ComputeLibrary"
     os.makedirs(acl_install_dir)
-    check_call(["git", "clone", "https://github.com/ARM-software/ComputeLibrary.git", "-b", "v23.08",
-                "--depth", "1", "--shallow-submodules"])
-    check_call(["scons", "Werror=1", "-j8", f"build_dir=/{acl_install_dir}/build"] + acl_build_flags,
-               cwd=acl_checkout_dir)
+    check_call(
+        [
+            "git",
+            "clone",
+            "https://github.com/ARM-software/ComputeLibrary.git",
+            "-b",
+            "v23.08",
+            "--depth",
+            "1",
+            "--shallow-submodules",
+        ]
+    )
+    check_call(
+        ["scons", "Werror=1", "-j8", f"build_dir=/{acl_install_dir}/build"]
+        + acl_build_flags,
+        cwd=acl_checkout_dir,
+    )
+    check_call(
+        [
+            "git",
+            "clone",
+            "https://github.com/ARM-software/ComputeLibrary.git",
+            "-b",
+            "v23.08",
+            "--depth",
+            "1",
+            "--shallow-submodules",
+        ]
+    )
+    check_call(
+        ["scons", "Werror=1", "-j8", f"build_dir=/{acl_install_dir}/build"]
+        + acl_build_flags,
+        cwd=acl_checkout_dir,
+    )
     for d in ["arm_compute", "include", "utils", "support", "src"]:
         shutil.copytree(f"{acl_checkout_dir}/{d}", f"{acl_install_dir}/{d}")
 
 
 def complete_wheel(folder: str) -> str:
-    '''
+    """
+    """
     Complete wheel build and put in artifact location
-    '''
+    """
+    """
     wheel_name = list_dir(f"/{folder}/dist")[0]
 
     if "pytorch" in folder:
         print("Repairing Wheel with AuditWheel")
-        check_call(["auditwheel","repair", f"dist/{wheel_name}"], cwd=folder)
+        check_call(["auditwheel", "repair", f"dist/{wheel_name}"], cwd=folder)
+        check_call(["auditwheel", "repair", f"dist/{wheel_name}"], cwd=folder)
```

Comment:
This seems to be duplicated

Reply:
code is a bit messy in this PR due to rebase, will open a new PR with cleaned up code